mcda/utils/utils_for_main.py

```diff
@@ -47,36 +47,41 @@ def check_config_error(condition: bool, error_message: str):
 
 def check_config_setting(condition_robustness_on_weights: bool,
                          condition_robustness_on_indicators: bool,
-                         mc_runs: int) -> (int, int):
+                         mc_runs: int, random_seed: int) -> (int, int):
     """
-        Checks configuration settings and logs information messages.
+    Checks configuration settings and logs information messages.
 
-        Parameters:
-        - condition (bool): The condition to check.
+    Parameters:
+    - is_robustness_weights, is_robustness_indicators, booleans indicating if robustness is considered
+    on weights or indicators.
 
-        Returns:
-        :rtype: Tuple[int, int]
-        :return: is_robustness_weights, is_robustness_indicators, a tuple containing flags indicating robustness
-                 on weights and indicators.
+    Example:
+    ```python
+    is_robustness_weights, is_robustness_indicators = check_config_setting(True, False, 1000, 42)
+    ```
 
-        Example:
-        ```python
-        is_robustness_weights, is_robustness_indicators = check_config_setting(True, "False,
-                                                                               "True", "False", 1000)
-        ```
-        """
+    :rtype: Tuple[int, int]
+    :return: (is_robustness_weights, is_robustness_indicators)
+    :param condition_robustness_on_weights: bool
+    :param condition_robustness_on_indicators: bool
+    :param mc_runs: int
+    :param random_seed: int
+
+    """
 
     is_robustness_weights = 0
     is_robustness_indicators = 0
 
     if condition_robustness_on_weights:
         logger.info("ProMCDA will consider uncertainty on the weights.")
         logger.info("Number of Monte Carlo runs: {}".format(mc_runs))
+        logger.info("The random seed used is: {}".format(random_seed))
         is_robustness_weights = 1
 
     elif condition_robustness_on_indicators:
         logger.info("ProMCDA will consider uncertainty on the indicators.")
         logger.info("Number of Monte Carlo runs: {}".format(mc_runs))
+        logger.info("The random seed used is: {}".format(random_seed))
```

Comment:
Duplicate log statement when compared to the if condition. This will later give you problem during debugging. Also, same applies to `Monte carlo` log statement.

Reply:
For the moment, I'd like to leave them like they are, if you agree.